Library/ViewModels/DiscoveryLightsOnEditorialViewModel.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import KsApi
+import ReactiveSwift
+
+public struct DiscoveryLightsOnEditorialCellValue: Equatable {
+  public let title: String
+  public let subtitle: String
+  public let imageName: String
+}
+
+public protocol DiscoveryLightsOnEditorialViewModelInputs {
+  func configureWith(_ value: DiscoveryLightsOnEditorialCellValue)
+}
+
+public protocol DiscoveryLightsOnEditorialViewModelOutputs {
+  var imageName: Signal<String, Never> { get }
+  var subtitleText: Signal<String, Never> { get }
+  var titleText: Signal<String, Never> { get }
+}
+
+public protocol DiscoveryLightsOnEditorialViewModelType {
+  var inputs: DiscoveryLightsOnEditorialViewModelInputs { get }
+  var outputs: DiscoveryLightsOnEditorialViewModelOutputs { get }
+}
+
+public final class DiscoveryLightsOnEditorialViewModel: DiscoveryLightsOnEditorialViewModelType,
```

Comment:
This is also the same as `DiscoveryEditorialViewModel` (should be `DiscoveryEditorialCellViewModel`), I think there is opportunity to reuse this code.